gnomad/utils/file_utils.py

```diff
@@ -85,6 +86,17 @@ async def unapplied_function():
     return dict(zip(fnames, file_existence))
 
 
+def parallel_file_exists(fnames: List[str]) -> Dict[str, bool]:
+    """
+    Call `parallel_file_exists_async` to check whether large number of files exist.
+
+    :param fnames: List of file names to check.
```

Comment:
Would add a sentence on whether these can be local or gs:// file paths (or both). 
Also maybe rename fnames => fpaths?